presto-main/src/test/java/com/facebook/presto/operator/scalar/AbstractTestFunctions.java

```diff
@@ -79,6 +80,18 @@ protected void assertDecimalFunction(String statement, SqlDecimal expectedResult
                 expectedResult);
     }
 
+    protected void assertThrowsPrestoException(String projection, StandardErrorCode errorCode, String message)
```

Comment:
In the other place you called the very similar method `assert*Function*ThrowsPrestoException`